bin/stacks

```diff
@@ -305,7 +307,10 @@ def main():
         cf_conn.close()
 
     if args.subcommand == 'create' or args.subcommand == 'update':
-        config = load_config('config.yaml', args.env)
+        if args.config is not None:
+            config = load_config(args.config.name, args.env)
```

Comment:
The argument is of type argparse.FileType, which is nice as then it gives you a sensible error when the file doesn't exist. But it means that args.config is a file, and your function takes a file name, hence adding .name to extract this.